src/cloud-init/microk8s/leader.yml.ts

```diff
@@ -174,14 +183,28 @@ const getLeaderCloudInitYaml = (config: CNDIConfig) => {
 
   const MICROK8S_ADD_NODE_TOKEN_TTL = 4294967295; //seconds 2^32 - 1 (136 Years)
 
+  let packages = ["apache2-utils", "nfs-common"];
+  let storageClassSetupCommands = [
+    `echo "Setting NFS as default storage class"`,
+    `while ! sudo microk8s kubectl patch storageclass nfs -p '{ "metadata": { "annotations": { "storageclass.kubernetes.io/is-default-class": "true" } } }'; do echo 'microk8s failed to install nfs, retrying in ${MICROK8S_INSTALL_RETRY_INTERVAL} seconds'; sleep ${MICROK8S_INSTALL_RETRY_INTERVAL}; done`,
+    `echo "NFS is now the default storage class"`,
+  ];
+
+  if (isDevCluster(config)) {
+    packages = ["apache2-utils"]; // no nfs-common on dev clusters
+    storageClassSetupCommands = [
+      `echo "Setting hostpath-storage as default storage class"`,
+      // TODO: fix the next 2 lines
```

Comment:
dev mode should set hostpath-storage as the storage class

can you handle that for me @IamTamika ?